### PR TITLE
[MTDSA-7950] amend loss claims typeOfClaim alignment

### PR DESCRIPTION
**Pull request by james-work-account**

### CODE REVIEW

File: resources/public/api/conf/1.0/schemas/lossClaims/amendClaim/amend_request.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "properties": {
     "typeOfClaim": {
       "type": "string",
-      "description": "The type of claim to be applied. A self employment loss claim can only be of type carry-forward or carry-sideways",
+      "description": "The type of claim to be applied.  Permitted values for \"self-employment\": carry-forward, carry-sideways.  Permitted values for \"uk-property-non-fhl\" and \"foreign-property\": carry-sideways, carry-sideways-fhl, carry-forward-to-carry-sideways.",
```
Comment (DreamcastKnight) [inline resources/public/api/conf/1.0/schemas/lossClaims/amendClaim/amend_request.json:9]: mentions foreign property here, afaik we don't have foreign property as a type of loss in V1?

File: resources/public/api/conf/1.0/schemas/lossClaims/amendClaim/amend_response.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     },
     "typeOfClaim": {
       "type": "string",
-      "description": "The type of claim to be applied. A self employment loss claim can only be of type carry-forward or carry-sideways.",
+      "description": "The type of claim to be applied.  Permitted values for \"self-employment\": carry-forward, carry-sideways.  Permitted values for \"uk-property-non-fhl\" and \"foreign-property\": carry-sideways, carry-sideways-fhl, carry-forward-to-carry-sideways.",
```
Comment (DreamcastKnight) [inline resources/public/api/conf/1.0/schemas/lossClaims/amendClaim/amend_response.json:22]: same as the previous comment

**Review by DreamcastKnight — CHANGES_REQUESTED** [review]

File: resources/public/api/conf/1.0/schemas/lossClaims/amendClaim/amend_request.json (hunk 1 above)
Comment (james-work-account) [reply resources/public/api/conf/1.0/schemas/lossClaims/amendClaim/amend_request.json:9]: Good spot, BA has confirmed that references to foreign-property can be removed from v1 - I'll do that now.

File: resources/public/api/conf/1.0/schemas/lossClaims/amendClaim/amend_response.json (hunk 2 above)
Comment (james-work-account) [reply resources/public/api/conf/1.0/schemas/lossClaims/amendClaim/amend_response.json:22]: ditto

**Review by DreamcastKnight — APPROVED** [review]

**Pull request merged**